assets/src/components/detours/detourMap.tsx

```diff
@@ -269,7 +269,7 @@ export const DetourMap = ({
             <>
               {uniqBy(stops, (stop) => stop.id).map((stop) => (
                 <StopMarkerWithStopCard
-                  key={stop.name}
+                  key={stop.id}
```

Comment:
Can you add the missed stop `index` to this key? Until we have `stop_sequence` this could also error on routes like (route pattern) `217-1-0`

https://www.mbta.com/schedules/217/line?from=global-search&query=217&schedule_direction%5Bdirection_id%5D=0&schedule_direction%5Bvariant%5D=217-1-0

![image](https://github.com/user-attachments/assets/01ddbc34-4cbc-4065-a27e-ff33c988e7a9)

I created a Detour in Skate Dev which shows this.

Reply:
I've checked out the skate dev detour, but I don't quite understand how this would error if key was changed to stop.id. I made the same detour locally with no error.  

Also, were you looking for an example where a stop is hit multiple times on the same route? I don't think this is one. There is a loop, but stops are not revisited

Local screenshot matches skate-dev, no console errors
![Screenshot 2024-10-22 at 8 22 16 AM](https://github.com/user-attachments/assets/9c688068-a83c-4362-9b4b-953d21481034)

Reply:
ooooh I thought this was a different part of the code, thanks for pointing that out. This change is good then 👍🏻.

---

It's `missed stops` that I was thinking of, which produces wrong results in production.

Notice that `North Quincy` is only listed once.

## Dev:
![image](https://github.com/user-attachments/assets/8ee53ed6-a37b-4546-86cf-274f8197aa61)

## Local patch:
![image](https://github.com/user-attachments/assets/9d70971e-8db3-4ee1-97d3-fc02cb081c97)

Reply:
Ohhhh I see. That's a good bug. I'm curious whether design would say the stop should be listed twice, or the missed stop count should be 7. Because the redundancy doesn't _seem_ like it would be valuable, but not certain

Reply:
I personally think it's valuable to riders, because it's possible that the stop could be missed on the first or second pass, but not the other. I think this also matters for GTFS-RT, so obscuring the link between operations data and rider data doesn't seem worth it to me.